.versionrc.js

```diff
@@ -0,0 +1,16 @@
+// .versionrc.js
+module.exports = {
+    bumpFiles: [
+        {
+            filename: "package.json",
+            type: "json"
+        },
+        {
+            filename: "dist/package.json",
+            type: "json"
+        }
+    ],
+    scripts: {
+        posttag: "copyfiles CHANGELOG.md dist/"
```

Comment:
this shouldn't be necessary any more -> CHANGELOG.md must be included in the files array in package.json

Reply:
thanks for the hint → removed